### DISCUSSION TITLE
Am I mis-remembering that it populates your media?

### DISCUSSION THREAD

**Discussion by fliberdygibits** [sbondCo/Watcharr#551] [Q&A] [answered]
I played with Watcharr months ago then got sidetracked.  Now I'm setting it up more properly on a new server.  I could swear I remembered previously that once you logged in to your JF acct it would populate all the media available but it seems like you have to know what shows are available and search for them before viewing or rating them.  Was I just crazy all those months ago?

**Comment by IRHM** [comment, marked as answer]
Hey @fliberdygibits, sadly you have to go to your profile page and click the 'Sync with Jellyfin' button (face menu > profile > sync button at bottom of page). This should be automated in the future.

Confirmed not crazy, thanks for using Watcharr!

**Reply by fliberdygibits** [reply]
Gack!  There it is!  I must have looked at it 20 times:)  Thank you!

**Reply by IRHM** [reply]
No problem!

**Reply by fliberdygibits** [reply]
Ok, perhaps I spoke too soon.  It works for a moment then reports no errors and that it imported the current list but nothing appears.  I'm poking at it now to see what I can sort out.

**Reply by IRHM** [reply]
@fliberdygibits oh that is sad, if it doesn't show after a full page refresh, I would recommend turning on debug logging in server settings and trying to sync again, there should hopefully be more useful info in the log then (if there isn't already a useful error in the log to look into).

**Reply by fliberdygibits** [reply]
I turned on debug logging and am not seeing anything... at least nothing which catches my untrained eye.    The last few new log entries since I ran a re-scan:

https://pastebin.com/9X5uF93T

**Reply by IRHM** [reply]
@fliberdygibits I also see nothing, any related logs should show up starting with `jellyfinSyncWatched:`. It looks like it is creating the job though, but nothing else which is odd. Are you on the latest version of watcharr?

**Reply by fliberdygibits** [reply]
Yep, latest version just downloaded and set up in the past few days.

**Reply by IRHM** [reply]
@fliberdygibits What does your jellyfin host setting look like? Do you host under a subdomain, a path on a domain or do you access jellyfin directly?
